tests/test_switch.py

```
@@ -12,6 +13,9 @@
 if TYPE_CHECKING:
     from homeassistant.core import HomeAssistant
 
+from custom_components.unifi_insights.api.network.models.routes import (
+    PolicyBasedRoute,
+)
```

Pylint (4.0.7)

[convention] Import "from custom_components.unifi_insights.api.network.models.routes import PolicyBasedRoute" should be placed at the top of the module

(C0413)